### ISSUE TITLE
Add extra columns in star files for Relion duplicate removal

### ISSUE THREAD

**Issue by mathewmclaren** [warpem/warp#272]
Hi all, 

Just wanted to suggest a small QOL improvement for processing Relion 5 data. Duplicate removal for tomography in Relion 5.0 (beta-3-commit-b978db) requires the columns rlnCenteredCoordinateX/Y/ZAngst in the run_data.star file to correctly locate the particles. I currently have to add them manually with relion_star_handler or a Python script.
I'm not well-versed enough in C# or submitting pull requests in Github, but I assume it would just require three lines additions [here](https://github.com/warpem/warp/blob/6c2ae96a68ee715cb17f1329e6110afe2768471a/Warp/Controls/TaskDialogs/Tomo/DialogTomoParticleExport.xaml.cs#L413)?I'm currently sampling particles on spheres so it's useful for me to remove them when the positions drift!

Cheers,

Mat

**Comment by alisterburt** [comment]
right, I agree you could add it but having the same information twice in different formats is asking for trouble and I don't think it's worth it for the minor quality of life improvement - backwards compatibility is a good point and worth keeping in mind
